src/drt/src/pa/FlexPA_acc_point.cpp

```diff
@@ -434,74 +447,66 @@ void FlexPA::genAPsFromRect(std::vector<std::unique_ptr<frAccessPoint>>& aps,
     }
   }
 
-  if (is_macro_cell_pin && use_center_line && is_layer1_horz) {
-    lower_type = frAccessPointEnum::OnGrid;
+  if (use_center_line && is_layer1_horz) {
+    for (auto& [layer1_coord, cost] : layer1_coords) {
+      layer1_coords[layer1_coord] = frAccessPointEnum::OnGrid;
+    }
```

Comment:
@bnmfw how is this equivalent?
This won't produce the same behavior for this check:
https://github.com/bnmfw/OpenROAD_Bernardo_Sandoval/blob/c0f5e592edb87dd99f80d17e14be4f74fe338b60/src/drt/src/pa/FlexPA_acc_point.cpp#L337
Also this for loop is duplicate to the else block of the previous if condition.